data/virt_autotest/alp-nat-vnet.xml

```diff
@@ -0,0 +1,20 @@
+<network>
+  <!--name can't have vnet keyword, to not conflict with virtual network test-->
+  <name>test-virt-net</name>
+  <uuid>cd1d0029-4ff9-4a24-9a3d-9f0877cb8477</uuid>
+  <forward mode='nat'>
+    <nat>
+      <port start='1024' end='65535'/>
+    </nat>
+  </forward>
+  <bridge name='br123' stp='on' delay='0'/>
+  <mac address='52:54:00:ea:a8:7e'/>
+  <ip address='192.168.123.1' netmask='255.255.255.0'>
+    <dhcp>
+      <range start='192.168.123.2' end='192.168.123.254'>
+        <lease expiry='12' unit='hours'/>
+      </range>
+    </dhcp>
+  </ip>
+</network>
+
```

Comment:
Please put all your configs in a ALP folder which is better.

Reply:
I considered this, but it is put here because it is more reasonable -- this config although used by alp, but it is  open to be used by other product's test too, after openqa guest installation code is modified to support other network types like virtual network.

Reply:
Then you should use other names instead of alp-nat-vnet.xml. You can use just some general names like vm-nat-vnet.xml or other names. And please be aware that when a same category of things keep growing, you should move them to a separate folder. I think networking config files belong to this category. That will be too late if we do not starting do so at some point.

Reply:
I thought about general names like  vm-nat-vnet.xm. In automation, it is open to use other virtual network config. If someone picks other config, and name the file specially, then how would we interpret this vm-nat-vnet.xm? So I name it alp-nat-vnet.xml, which means it is the customized vnet config file for alp, although it is capable to be reused in other products like sles. Make sense? 
Besides, in reality, I think we won't have too much such virtual network config files, at most several. And Leon's several [network].xml stay static like that(won't grow for years). Do you think it is necessary to add a sub-dir for several files? Personally I prefer not. 

But if you insist on either item, I can change.

Reply:
I do not insist on the names. I think you should put your config files into a separate folder. Just like I said above, starting take actions to organize them better before it is too late. Putting everything in top folder is not very good. For example, a virtual_network_config folder or other names you prefer. We should start getting into habit of organizing files better. Think about it.

Reply:
Okay, will create separate folder for virtual_network_config. This is good direction anyway.

Reply:
done